src/styles/blocks/catalog.scss

```
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: $transition-body;
```

By default transition is applienig to all - thats is hard for browser. 
Your should set only values you change.
Fix same a bit bottom
```suggestion
    transition: color $transition-duration, transform $transition-duration;
```